docs/examples/userguide/language_basics/casting_python.pyx

```diff
@@ -1,8 +1,5 @@
 from cpython.ref cimport PyObject
 
-cdef extern from *:
```

Comment:
Can we delete `casting_python.pxd` too? I think it essentially duplicates this unnecessary declaration

Reply:
yes, absolutely!

Reply:
I think keep the pxd file. I slightly misunderstood the problem here.

Reply:
One more change here (sorry):

If the pxd file is being used in the pyx example, then it should be shown in  the "Cython" tab too (and not just the "Pure Python" tab).

Should be added here I think:
https://github.com/cython/cython/blob/c2d9ab2ebc9ef61ecb7adffe311c930aecb99b7b/docs/src/userguide/language_basics.rst?plain=1#L1257

Reply:
There is just one problem with running all examples with `cython -3`, last week the bind [`_Py_REFCNT` was added](https://github.com/cython/cython/pull/6152) and I didn't find any mention of this in the change log to the next release. 

So, running [`parameter_refcount`](https://github.com/cython/cython/blob/master/docs/examples/userguide/language_basics/parameter_refcount.pyx#L1) examples the output was: `'cpython/ref/_Py_REFCNT.pxd' not found`.  Is that something that I should have in my local env?

Reply:
Thanks

> There is just one problem with running all examples with `cython -3`, last week the bind [`_Py_REFCNT` was added](https://github.com/cython/cython/pull/6152) and I didn't find any mention of this in the change log to the next release.
> 
> So, running [`parameter_refcount`](https://github.com/cython/cython/blob/master/docs/examples/userguide/language_basics/parameter_refcount.pyx#L1) examples the output was: `'cpython/ref/_Py_REFCNT.pxd' not found`. Is that something that I should have in my local env?

I think this is just because you're testing these with an older version of Cython that doesn't have the change. The easiest way to fix it is probably to directly call the cython.py from the git repo instead of calling "cython" (which links to whatever's installed).

Reply:
ah ok! Thanks.